docs/admin/kubeadm.md

```diff
@@ -168,20 +168,71 @@ necessary.
 By default, when `kubeadm init` runs, a token is generated and revealed in the output.
 That's the token you should use here.
 
+
+## Using kubeadm with a config file
+
+It's possible to configure kubeadm with a config file instead of command line flags, and the more advanced modes of kubeadm will only be
```

Comment:
Suggest: "and some more advanced features may only be available as configuration file options", and replacing "config" with "configuration".